schema/v5.0/CVE_JSON_5.0.schema

```diff
@@ -247,54 +258,72 @@
                     "uniqueItems": true,
                     "items": {
                         "type": "object",
-                        "description": "Affected/non-affected/fixed versions of a given technology, product, hardware, etc.",
-                        "required": ["versionValue"],
+                        "description": "A single version or a range of versions, with vulnerability status.\n\nAn entry with only 'version' and 'status' indicates the status of a single version.\n\nOtherwise, an entry must include the 'range' property, to define the version numbering semantics in use, and 'limit', to indicate the non-inclusive upper limit of the range. The object describes the status for versions V such that 'version' <= V and V < 'limit', using the <= and < semantics defined for the specific kind of 'range'. Status changes within the range can be specified by an optional 'changes' list.\n\nThe algorithm to decide the status specified for a version V is:\n\n\tfor entry in versions {\n\t\tif entry.limit is not present and v == entry.version {\n\t\t\treturn entry.status\n\t\t}\n\t\tif entry.limit is present and v <= entry.version and v < entry.limit { // <= and < defined by entry.range\n\t\t\tstatus = entry.status\n\t\t\tfor change in entry.changes {\n\t\t\t\tif v >= change.at {\n\t\t\t\t\tstatus = change.status\n\t\t\t\t}\n\t\t\t}\n\t\t\treturn status\n\t\t}\n\t}\n\treturn \"unknown\"\n\n",
+                        "oneOf": [
+                            {
+                                "required": ["version", "status"],
+                                "maxProperties": 2
+                            },
+                            {
+                                "required": ["version", "status", "range", "limit"]
+                            }
+                        ],
                         "properties": {
-                            "versionGroup": {
-                                "type": "string",
-                                "description": "A string that represents a version branch, group, or a major version (e.g. 10.0, 3.1.*) where all version values are typically sequential or versionAffected comparisons are meaningful (optional).",
-                                "minLength": 1,
-                                "maxLength": 1024
+                            "version": {
+                                "description": "The single version being described, or the version at the start of the range.",
+                                "$ref": "#/definitions/version"
                             },
-                            "versionValue": {
+                            "status": {
+                                "description": "The vulnerability status for the version or range of versions. For a range, the status may be refined by the 'changes' list.",
+                                "$ref": "#/definitions/status"
+                            },
+                            "range": {
```

Comment:
Is there a better name for "range"? how about "versionType"?

Since `range : "1.2 thru 1.5"` reads perfectly fine, I don't want people to enter wrong data here.

Reply:
Done.